modules/gapi/include/opencv2/gapi/imgproc.hpp

```diff
@@ -871,6 +927,70 @@ Output image must be 8-bit unsigned 3-channel image @ref CV_8UC3.
 */
 GAPI_EXPORTS GMat RGB2YUV(const GMat& src);
 
+/** @brief Converts an image from BGR color space to I420 color space.
+
+The function converts an input image from BGR color space to I420.
+The conventional ranges for R, G, and B channel values are 0 to 255.
+
+Output image must be 8-bit unsigned 1-channel image. @ref CV_8UC1.
+Width of I420 output image must be the same as width of input image.
+Height of I420 output image must be equal 3/2 from height of input image.
```

Comment:
> 3/2 from height

Why is not tuple of `Y` and `UV`?

/cc @TolyaTalamanov (implemented drawing over NV12 frames)

Reply:
@alalek tuple of Y and UV is what we'll do next, but the current request is to provide a `cvtColor`-based counterpart.

Reply:
Please note that there is no C++ overloads for return type only. So it will be hard to improve that without breaking of existed code.
Just keep in mind.

Reply:
ok